### PR TITLE
Dutch translations

### PR DESCRIPTION
**Pull request by reyostallenberg**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Fixed issue   | -
| License       | MIT

Add translation file for Dutch

### CODE REVIEW

File: src/Resources/translations/messages.nl.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            last_return_code: Laatste retourcode
+            log_file: Log-bestand
+            priority: Prioriteit
+            enabled: Ingeschakeld
+            command_execution_time: Uitvoer duur
+            execute_immediate: Voer direct uit
+            emtpy_logs: Logs legen
+            bulk_empty_logs: De log-bestanden zijn geleegd.
+        edit_scheduled_command: Bewerk commando
+        new_scheduled_command: Creëreer commando
```
Comment (reyostallenberg) [inline src/Resources/translations/messages.nl.yml:19]: Fix typo
```suggestion
        new_scheduled_command: Creëer commando
```

**Comment by oallain** [comment]
Hello,

Thanks for that :)

**Comment by oallain** [comment]
@reyostallenberg 🖖🏼 

You can use it with new [version 2.2.0](https://github.com/synolia/SyliusSchedulerCommandPlugin/releases/tag/v2.2.0) 🎉

**Comment by reyostallenberg** [comment]
That's awesome, thank you

**Pull request merged**